docs/DeveloperGuide.adoc

```diff
@@ -358,25 +420,28 @@ _{More to be added}_
 [[mainstream-os]] Mainstream OS::
 Windows, Linux, Unix, OS-X
 
-[[private-contact-detail]] Private contact detail::
-A contact detail that is not meant to be shared with others
+[[eatery]] Eatery::
+Any place that serves food open to the public (ie, restaurants, cafes, roadside, pushcart)
+
+[[category]] Category::
+Broadly containts cuisines (self-explanatory) and eating place (ie, a buffet, fusion)
```

Comment:
May want to define what "todo" means as developers may need to understand such terms :)